RELEASE-NOTES.md

```diff
@@ -32,6 +32,7 @@
 * Added support for user stages in stage execute command
 * Added support for user stages in stage and git copy commands
 * Improved support for quoted identifiers in snowpark commands.
+* Updated post_install SQL script default database to be the application database
```

Comment:
Isn't it called post_deploy from the POV of the user?

Reply:
updated.